aioredis/commands/streams.py

```diff
@@ -63,10 +67,16 @@ def parse_messages_by_stream(messages_by_stream):
     """
     if messages_by_stream is None:
         return []
-
     parsed = []
+
     for stream, messages in messages_by_stream:
```

Comment:
I think I'd like to have two branches here - one that handles decoding and one that does not, so that we don't have to re-evaluate the existence of an encoding parameter for every message returned.